thoth/storages/graph/postgres.py

```diff
@@ -3693,6 +3693,51 @@ def get_kebechet_github_installations_info_for_python_package_version(
 
         return self._group_by_slug(result)
 
+    def update_kebechet_installation_using_files(
+        self,
+        slug: str,
+        requirements: Optional[dict] = None,
+        requirements_lock: Optional[dict] = None,
+        thoth_config: Optional[dict] = None,
+    ):
+        """Update info about kebechet installation."""
+        with self._session_scope() as session, session.begin(subtransactions=True):
+            if "runtime_environments" in thoth_config:
+                env_dict = thoth_config["runtime_environments"][0]
```

Comment:
Do we want to remove adviser sync part that store kebechet? https://github.com/thoth-station/storages/blob/b29d1606d5dce208e18012e456900810293284ed/thoth/storages/graph/postgres.py#L5361

Reply:
> what we do for users that have more than one software stack? overlays? it is a feature request for kebechet in any case I guess, we can not updat on overlays yet. Let's add a TODO: there to remind we need to make that a for loop on all runtime environments present.

This will basically just require adding a `HasSoftwareEnv` table for the relationship instead of having a single column.  How would we handle advise? should we advise on all software environments too?